edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj

```diff
@@ -3,8 +3,10 @@
     <CompileBlocked>true</CompileBlocked>
   </PropertyGroup>
 
-  <Import Project="..\..\..\..\..\netstandardVersion.props" />
-
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Can you call out why you're pinning this project and `EdgeHubTriggerCSharp` to netstandard2.0?

Reply:
it breaks on dotnet 2.0 b/c its dependencies aren't updated

Reply:
I meant can you call it out in a code comment or in the PR description.

Reply:
sure